### DISCUSSION TITLE
Protect directory/file

### DISCUSSION THREAD

**Discussion by gtirloni** [community/community#16441] [Pull Requests]
Protected branches are great but sometimes we have a specific directory/file in a repository that needs an extra set of eyes every time there is a change.

The CODEOWNERS file syntax could be improved to accept something like `approvals=N`.

**Comment by Boogieman2022** [comment]
> Protected branches are great but sometimes we have a specific directory/file in a repository that needs an extra set of eyes every time there is a change.
> 
> The CODEOWNERS file syntax could be improved to accept something like `approvals=N`.

Wouldn't an approval =Y`?

**Reply by gtirloni** [reply]
I didn't get what you mean. I'm proposing we require additional approvals (N>1).

**Comment by fellyciasutanto** [comment]
Hello, I have the same question. Were you able to figure this out?

**Comment by EwoutH** [comment]
I would really love such a feature! It's insane we can protect branches but not files or directories.
